src/main/java/com/exasol/adapter/dynamodb/mapping/AbstractPropertyToColumnMapping.java

```diff
@@ -42,4 +42,29 @@ public final DocumentPathExpression getPathToSourceProperty() {
     public final LookupFailBehaviour getLookupFailBehaviour() {
         return this.lookupFailBehaviour;
     }
+
+    @Override
+    public boolean equals(final Object other) {
+        if (this == other) {
+            return true;
+        }
+        if (!(other instanceof AbstractPropertyToColumnMapping)) {
+            return false;
+        }
+        if (!super.equals(other)) {
+            return false;
+        }
+        final AbstractPropertyToColumnMapping that = (AbstractPropertyToColumnMapping) other;
+        if (!this.pathToSourceProperty.equals(that.pathToSourceProperty)) {
+            return false;
+        }
+        return this.lookupFailBehaviour == that.lookupFailBehaviour;
```

Comment:
It is usually good practice easy (primitive types before arrays for example) comparison to put first.
```suggestion
        return this.lookupFailBehaviour == that.getLookupFailBehaviour() 
                  && this.getExasolColumnName().equals(that.getExasolColumnName()) 
                  && this.pathToSourceProperty.equals(that.getPathToSourceProperty());
```